server.go

```diff
@@ -491,6 +491,8 @@ func NewFromConfig(config ServerConfig) (*Server, error) {
 		Workers: make([]*Worker, max(1, conf.NumWorkers)),
 	}
 
+	ret.Tags = append(ret.Tags, VERSION)
```

Comment:
We do not want `VERSION` attached to all tags, just the heartbeat metric itself.

Reply:
Make sense!